code/modules/shuttle/medisim.dm

```diff
@@ -0,0 +1,62 @@
+///These are for the medisim shuttle
+
+/obj/machinery/capture_the_flag/medisim
+	game_area = /area/shuttle/escape
+
+/obj/machinery/capture_the_flag/medisim/spawn_team_member(client/new_team_member)
+	var/mob/living/carbon/human/M = ..()
+	M.remove_all_languages(LANGUAGE_CTF)
+	M.grant_language(/datum/language/monkey, TRUE, TRUE, LANGUAGE_CTF)
+
+//maybe add something here about ending the other CTF game/not preventing that from happening
+
+//late initialize for hooking up the computers on the shuttle
+
+/obj/machinery/capture_the_flag/medisim/reset_the_arena()
+	return //so we don't delete every object on the shuttle. downsides = that simulation room is gonna get gross pretty quick
```

Comment:
It'd be a little extra work, but you could achieve this with 3 landmarks defining the corners of the 2 rectangles of turfs you'd need to clear

Reply:
Probably not necessary as the shuttle will only be used for like 5 minutes though

Reply:
fixed by using a subtype'd shuttle area!

Reply:
oh nice, didn't realize you could do that lol